src/libpsl-native/src/geterrorcategory.cpp

```diff
@@ -0,0 +1,27 @@
+#include "geterrorcategory.h"
+
+#include <assert.h>
+#include <sys/types.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+//! @brief Maps Linux errno to PowerShell ErrorCategory
```

Comment:
I'd just add the full enum in C - that will make these "references" more obvious that they're the same as the C# code.